app/assets/javascripts/discourse/app/components/group-selector.js

```diff
@@ -25,6 +25,7 @@ export default Component.extend({
     let groupNames = this.groupNames;
 
     $(this.element.querySelector("input")).autocomplete({
+      debounced: true,
```

Comment:
I'm wondering if this should be opt-ed out instead so that we have something that is performance optimized by default

Reply:
I suggest we try like this for a few weeks, and then maybe change the default